BunqSdk/Security/SecurityUtils.cs

```diff
@@ -127,6 +133,20 @@ private static string GenerateRequestHeadersSortedString(HttpRequestMessage requ
             );
         }
 
+        private static string GetHeaderNameCorrectyCased(string headerName)
```

Comment:
GetHeaderName**Correcty**Cased

Reply:
Lets create a follow up issue to add tests in all SDK's. I've done a quick dirty test by manipulating the headers right after the response is received to see if it indeed works, which it does. But a test unit is indeed needed to ensure that in the future it does not break.

Reply:
bunq/sdk_csharp#60